tests/e2e/labelupdates.go

```diff
@@ -421,8 +421,8 @@ var _ bool = ginkgo.Describe("[csi-block-vanilla] label-updates", func() {
 		err = client.CoreV1().PersistentVolumeClaims(namespace).Delete(ctx, pvc.Name, *metav1.NewDeleteOptions(0))
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		// Waiting for some time for PVC to be deleted correctly
-		ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow PVC deletion", sleepTimeOut))
-		time.Sleep(time.Duration(sleepTimeOut) * time.Second)
+		ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow PVC deletion", oneMinuteWaitTimeInSeconds))
```

Comment:
can we add polling? We are trying to replace all the sleeps in the code.

Reply:
In this case, you cannot! Since the verification is on the CNS side we cannot add polling in CSI/k8s environment